new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -65,69 +64,89 @@ public void testInstallingOperator() {
+    // Create a service account for the unique opNamespace
+    final String serviceAccountName = opNamespace + "-sa";
+    assertDoesNotThrow(() -> TestActions.createServiceAccount(new V1ServiceAccount()
+        .metadata(new V1ObjectMeta().namespace(opNamespace).name(serviceAccountName))));
     logger.info("Created service account: " + serviceAccountName);
 
     OperatorParams opParams =
         new OperatorParams().releaseName("weblogic-operator")
-            .namespace(namespace)
-            .image("weblogic-kubernetes-operator:test_itsimpleoperator")
-            .domainNamespaces(Arrays.asList("domainns1", "domainns2"))
-            .serviceAccount("opns1-sa");
+            .namespace(opNamespace)
+            .chartDir("../kubernetes/charts/weblogic-operator")
+            .image("phx.ocir.io/weblogick8s/weblogic-kubernetes-operator:develop")
+            .domainNamespaces(Arrays.asList(domainNamespace1, domainNamespace2))
+            .serviceAccount(serviceAccountName);
 
-    //ToDO: use Junit 5 assertions
     try {
-      installOperator(opParams);
-    } catch (ApiException e) {
-      e.printStackTrace();
-      fail("Failed to install Operator due to exception" + e.getMessage());
-    }
+      assertThat(installOperator(opParams)).as("Operator installation failed").isTrue();
+      logger.info(String.format("Operator installed in namespace %s", opNamespace));
+
+      // we can use a standard JUnit assertion to check on the result
+      // assertEquals(true, success, "Operator successfully installed in opNamespace " + opNamespace);
+
+      // this is an example of waiting for an async operation to complete.
+      // after the previous step was completed, kubernetes will try to pull the image,
+      // start the pod, check the readiness/health probes, etc.  this will take some
+      // period of time and either the operator will come to a running state, or it
+      // will not.
+      // in this example, we first wait 30 seconds, since it is unlikely this operation
+      // will complete in less than 30 seconds, then we check if the operator is running.
+      with().pollDelay(30, SECONDS)
+          // we check again every 10 seconds.
+          .and().with().pollInterval(10, SECONDS)
+          // this listener lets us report some status with each poll
+          .conditionEvaluationListener(
+              condition -> logger.info(()
+                  -> String.format("Waiting for operator to be running (elapsed time %dms, remaining time %dms)",
+                  condition.getElapsedTimeInMS(),
+                  condition.getRemainingTimeInMS())))
+          // and here we can set the maximum time we are prepared to wait
+          .await().atMost(5, MINUTES)
+          // operatorIsRunning() is one of our custom, reusable assertions
+          .until(operatorIsRunning(opNamespace));
+    } finally {
+
+      // uninstall operator release
+      assertThat(TestActions.uninstallOperator(opParams))
```

Comment:
@rjeberhard @ddsharpe interested in your opinion on this --- seems like assertions in the finally is somehow wrong -- if these are important, should they not be part of the test itself?   not sure about assertions in a finally block.....

Reply:
Hmmm... I agree that that does seem incorrect.  If the point is to test uninstallation then this should be a test; otherwise, I'd have clean-up in an extension or "after" lifecycle method.

Reply:
moved to after lifecycle method.